### PR TITLE
[git] ignore vscode configuration files

### PR DESCRIPTION
**Pull request by wgtdkp**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -27,6 +27,7 @@
 .local-version
 .libs
 .vagrant
+.vscode
```
Comment (simonlingoogle) [inline .gitignore:30]:
```suggestion
.vscode/
```

Comment (wgtdkp) [reply .gitignore:30]: I am following the style above this line. @jwhui @simonlingoogle Do you think it is necessary to change them all?
```
.deps
.dirstamp
.DS_Store
.local-version
.libs
.vagrant
.vscode
```

**Review by abtink — COMMENTED** [review]
LGTM.

File: .gitignore (hunk 1 above)
Comment (simonlingoogle) [reply .gitignore:30]: I think we should move .vscode/ below (together with .idea/) because they are both IDE configuration files.

**Review by simonlingoogle — APPROVED** [review]
LGTM :+1: 

with nit

**Comment by wgtdkp** [comment]
@abtink I think you actually wanted to approve this PR :smile:

File: .gitignore [hunk 2]
```diff
@@ -60,7 +60,8 @@ third_party/nlbuild-autotools/repo/third_party/autoconf/m4/lt~obsolete.m4
 third_party/nlbuild-autotools/repo/third_party/autoconf/missing
 /tmp/
 
-# C-Lion / IDEA files
+# IDE / editor files
+.vscode/**
 .idea/**
```
Comment (jwhui) [inline .gitignore:65]: nit: alphabetical ordering?
```suggestion
.idea/**
.vscode/**
```

**Review by jwhui — APPROVED** [review]

**Pull request merged**